### DISCUSSION TITLE
What classes should a model predict - brats dataset

### DISCUSSION THREAD

**Discussion by amida47** [Project-MONAI/MONAI#4335] [Q&A] [answered]
I am trying to train a model for the segmentation task of brats dataset and
I am confused and can't figure out which classes to train my model to predict
should I train it to predict :
whole tumor , tumor core, enhancing
or
edema , enhancing, necrotic

**Comment by Nic-Ma** [comment, marked as answer]
Hi @amida47 ,

For Brats training, I think you can refer to the tutorial:
https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/brats_segmentation_3d.ipynb

Thanks.
